src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -62,6 +62,10 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <!-- dependencies coming from the package manager lock file should not be copied locally for .NET Core and .NETStandard projects -->
     <CopyLocalLockFileAssemblies Condition="'$(CopyLocalLockFileAssemblies)' == ''">false</CopyLocalLockFileAssemblies>
+
+    <!-- Disable the use of FrameworkPathOverride in Microsoft.Common.CurrentVersion.targets which can slow down evaluation.  FrameworkPathOverride
+    is not needed for NETStandard or NETCore since references come from NuGet packages-->
+    <UseFrameworkPathOverride>false</UseFrameworkPathOverride>
```

Comment:
We let the user/other targets opt out, can you condition it against itself?

Reply:
I've added a condition